Changelog.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## [unreleased]
+- Change name used in assignment grade distribution to grader's display name attribute (#5844)
```

Comment:
@therealfake I keep forgetting to mention this, but please add entries to the bottom of the section (trying to keep the entries in numerical order)